launcher/modManager/cmodmanager.cpp

```diff
@@ -18,7 +18,11 @@
 #include "../jsonutils.h"
 #include "../launcherdirs.h"
 
-static QString detectModArchive(QString path, QString modName)
+namespace
+{
+const QLatin1String extraResolutionsMod{"vcmi-extras.extraresolutions"};
```

Comment:
So if we have another mod, e.g. "android-resolutions" it won't work?

Reply:
yes, currently it's only about VCMI extras, just like I initially proposed. That's why proper check of all mods is needed.